assets/src/components/customers/CustomerDetailsModal.tsx

```diff
@@ -103,6 +103,14 @@ export const CustomerDetailsContent = ({customer}: {customer: any}) => {
         </Paragraph>
       </Box>
 
+      <Box mb={2}>
+        <Box>
+          <Text strong>Time zone</Text>
+        </Box>
+
+        <Paragraph>{time_zone || 'N/A'}</Paragraph>
```

Comment:
instead of "N/A" let's do "Unknown"

Reply:
changed position and null text in the latest commit. Sometimes, I tend to miss minor but important details in the problems😪

Reply:
no worries :)